timber/src/main/java/timber/log/Timber.java

```diff
@@ -345,13 +387,17 @@ private void throwShade(int priority, String message, Throwable t) {
       }
 
       String tag = createTag();
-      logMessage(priority, tag, message);
+      logMessage(priority, tag, message, t);
     }
 
     /** Log a message! */
-    protected void logMessage(int priority, String tag, String message) {
+    protected void logMessage(int priority, String tag, String message, Throwable throwable) {
```

Comment:
Why bother including the throwable? It's already appended to the log message and it's not passed to the WTF handler.